mixlib-shellout-x86-mingw32.gemspec

```diff
@@ -1,9 +0,0 @@
-# x86-mingw32 Gemspec #
```

Comment:
Yes I did - I was confused about how bundler was actually handling gemspecs.  Turns out that you can only point it to a single gemspec file in your Gemfile, but then what it does (either by design or otherwise) is that it loads all gemfiles located at that path regardless of the file name.  It then looks for all gemspec objects whose name attribute corresponds to the name provided in the Gemfile, filters out all gemspec objects that don't match the current platform and then "sorts" the list (I haven't figured out exactly how it sorts it) and picks the top match.  It's convoluted but this works with it.  It installs windows-y things on windows and posix-y things otherwise.